### PR TITLE
add(script): Added gitlab script for cstor csi snapshot and clone test cases

### PR DESCRIPTION
**Pull request by w3aman**
Signed-off-by: Aman Gupta <aman.gupta@mayadata.io>
Gitlab runner scrippt added for following test cases

- cspc-striped-pool
- csi-provisioner
- csi-snapshot
- csi-clone

### CODE REVIEW

File: script/2-infra-setup/deploy-csi-provisioner/csi-provisioner [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+commit_id=$CI_COMMIT_SHA
+releaseTag=$RELEASE_TAG
+source ~/.profile
+gittoken=$(echo "$github_token")
+
+time="date"
+current_time=$(eval $time)
+
+echo "Setting up test dependencies.."
+${utils_path}/setup_dependencies infra-setup
```
Comment (somesh2905) [inline script/2-infra-setup/deploy-csi-provisioner/csi-provisioner]: modify dependency to litmus test

File: script/3-functional/csi-cstor-clone/csi-cstor-clone [hunk 2]
```diff
@@ -0,0 +1,194 @@
...
+commit_id=$CI_COMMIT_SHA
+releaseTag=$RELEASE_TAG
+source ~/.profile
+gittoken=$(echo "$github_token")
+
+time="date"
+current_time=$(eval $time)
+
+echo "Setting up test dependencies.."
+${utils_path}/setup_dependencies infra-setup
```
Comment (somesh2905) [inline script/3-functional/csi-cstor-clone/csi-cstor-clone]: modify dependency to litmus test

**Review by somesh2905 — CHANGES_REQUESTED** [review]

File: script/2-infra-setup/deploy-csi-provisioner/csi-provisioner (hunk 1 above)
Comment (w3aman) [inline script/2-infra-setup/deploy-csi-provisioner/csi-provisioner:15]: Done

File: script/3-functional/csi-cstor-clone/csi-cstor-clone (hunk 2 above)
Comment (w3aman) [reply script/3-functional/csi-cstor-clone/csi-cstor-clone]: Done

File: script/3-functional/csi-cstor-snapshot/csi-cstor-snapshot [hunk 3]
```diff
@@ -0,0 +1,98 @@
...
+commit_id=$CI_COMMIT_SHA
+releaseTag=$RELEASE_TAG
+source ~/.profile
+gittoken=$(echo "$github_token")
+
+time="date"
+current_time=$(eval $time)
+
+echo "Setting up test dependencies.."
+${utils_path}/setup_dependencies infra-setup
```
Comment (somesh2905) [inline script/3-functional/csi-cstor-snapshot/csi-cstor-snapshot]: modify dependency to litmus test

**Review by somesh2905 — CHANGES_REQUESTED** [review]

**Comment by w3aman** [comment]
Not required further...
closing

**Pull request closed without merging**